src/membraneWebRTC.ts

```diff
@@ -785,15 +793,21 @@ export class MembraneWebRTC {
               encoding.rid! as TrackEncoding
             ) || 0;
 
-          if (limit > 0) encoding.maxBitrate = limit * 1024;
-          else delete encoding.maxBitrate;
+          if (limit > 0) {
+            encoding.maxBitrate = limit * 1024;
+            if (mediaEvent)
+              mediaEvent.data.data.variantBitrates[encoding.rid!] =
+                encoding.maxBitrate;
+          } else delete encoding.maxBitrate;
```

Comment:
This case will not be adequately handled on the server. I think what will happen is that the encoding will be missing from the media event and the server will still consider the previous limit valid, which is not true because the limit has been removed, so we're back to Chrome's default limit for the variant.

Maybe we should just force the limit values to be set for all present variants in the Media Event? :thinking: This way we won't have to hard code the default Chrome limits in the RTC Engine except for backward compatibility reasons. This approach may also be better when you consider other clients, which may have different default limits.